src/auto/tls_connection.rs

```diff
@@ -47,6 +47,9 @@ pub trait TlsConnectionExt: 'static {
 
     fn get_certificate(&self) -> Option<TlsCertificate>;
 
+    //#[cfg(any(feature = "v2_66", feature = "dox"))]
+    //fn get_channel_binding_data(&self, type_: /*Ignored*/TlsChannelBindingType, data: /*Ignored*/glib::ByteArray) -> Result<(), glib::Error>;
```

Comment:
That can be easily generated if you add the enum to the `generate` list and `GLib.ByteArray` to the `manual` list in `Gir.toml`.

Reply:
Done, adding GLib.ByteArray generated a bunch of other bindings. We should probably check the not bound ones & that could b generated in the future.